src/main/steps/applicant1/pay-and-submit/content.ts

```diff
@@ -15,13 +15,26 @@ const en = ({ partner }: CommonContent) => ({
   detailsHeading: 'If you cannot pay',
   line3: `The payment system will only allow you to pay, but you could talk to your ${partner} about whether they would be prepared to send you some money.`,
   line4:
-    'Or you could submit a sole application. If you apply as a sole applicant, only you have to apply for Help With Fees. You will not have to re-enter the information you have already provided. You will need to provide some new information though.',
-  line5: `You can <a href="${SWITCH_TO_SOLE_APPLICATION}" class="govuk-link">submit a sole application here</a>`,
+    'Or you could submit a sole application. If you apply as a sole applicant, only you have to apply for Help With Fees. You will not have to re-enter the information you have already provided but you will have to provide some new information.',
+  line5: `You can <a href="${SWITCH_TO_SOLE_APPLICATION}" class="govuk-link">create a new application here.</a>`,
   line6: 'This joint application will not be submitted until you pay the fee.',
   continue: 'Pay and submit',
 });
 
-const cy: typeof en = en;
+const cy: typeof en = ({ partner }: CommonContent) => ({
+  title: 'Talu a chyflwyno',
+  line1: `Mae eich cais ar y cyd wedi'i gytuno gennych chi a'ch ${partner}. Mae angen i chi dalu’r ffi am wneud cais, sef ${getFee(
+    config.get('fees.applicationFee')
+  )} cyn y gellir ei gyflwyno. Nid yw’r system dalu yn caniatáu i chi rannu’r taliad. `,
```

Comment:
```suggestion
  )} cyn y gellir ei gyflwyno. Nid yw’r system dalu yn caniatáu i chi rannu’r taliad.`,
```

Reply:
Thanks Daniel, yeah I removed the extra space.